iXBRLViewerPlugin/viewer/src/html/inspector.html

```diff
@@ -226,6 +226,7 @@ <h2 class="title"></h2>
     <div id="iframe-container">
       <div class="zoom-in">+</div>
       <div class="zoom-out">-</div>
+      <div class="print"><img alt="Print Document" src="../img/print.svg" /></div>
```

Comment:
The print icon is using `#333`, but the zoom icons are `#777`.   I don't have a preference, but we should be consistent.  

`#333`:
![image](https://user-images.githubusercontent.com/45077928/196540452-1c5b9c48-a365-40da-88ed-cfc7dfd36a7e.png)

`#777`:
![image](https://user-images.githubusercontent.com/45077928/196540503-93dd7400-b324-4671-9f24-9b219684e4f6.png)

I submit a PR to this PR in a moment which adds the print icon to the font file, which means that this can be set via CSS.

Reply:
Thank you, @paulwarren-wk for the update. I tried my hand at doing it myself and got close, but not that close.

Reply:
The build process for that font file is far from ideal, and relies on resources that aren't in this repo.  I've got an idea for improving it, which I'll put in a separate PR.